python/chronos/src/bigdl/chronos/utils.py

```diff
@@ -30,3 +31,33 @@ def wrapped(*args, **kwargs):
             return function(*args, **kwargs)
         return wrapped
     return deprecated_decorator
+
+
+class LazyImport:
+    """
+        code adaptted from https://github.com/intel/neural-compressor
+        /blob/master/neural_compressor/utils/utility.py#L62
+
+        :param module_name: The name of module imported later
+        :param pkg: prefix path.
+    """
+    def __init__(self, module_name, pkg=None):
+        self.module_name = module_name
+        self.pkg = pkg
+
+    def __getattr__(self, name):
+        try:
+            module = importlib.import_module(self.module_name, package=self.pkg)
+            mod = getattr(module, name)
+        except:
+            spec = importlib.util.find_spec(str(self.module_name + '.' + name))
```

Comment:
On what circumstances this `except` will be effective, I don't think we may find this "..LSTMForecaster.predict" or some thing like that

Reply:
you are right. `..LSTMForecaster` was originally designed to solve the problem of relative path import, but will cause the above problem, and now it has all been imported with absolute path.